src/interfaces/IVault.sol

```diff
@@ -1,11 +1,16 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+pragma solidity 0.8.26;
```

Comment:
I would not lock interfaces as many components could include with different version of solidity

Reply:
b4f68b66027810ab7280cf5bcb9d86532c9dd9e3